cdap-unit-test/src/test/java/co/cask/cdap/test/app/TestFrameworkTestRun.java

```diff
@@ -341,6 +343,20 @@ public void testDeployWorkflowApp() throws Exception {
     Assert.assertEquals(false, Boolean.parseBoolean(workflowToken.getTokenData().get("running").get(0).getValue()));
   }
 
+  @Test
+  public void testBatchStreamUpload() throws Exception {
+    StreamManager batchStream = getStreamManager("batchStream");
+    batchStream.createStream();
+    URL streamDataUrl = Thread.currentThread().getContextClassLoader().getResource("test-stream-data.txt");
```

Comment:
Better not to put a file in resource. You can use Temporary folder and generate a file with the desired content. This makes the file content and the assertion code together.